docs/en/observability/logs-checklist.asciidoc

```diff
@@ -2,78 +2,89 @@
 = Logs resource guide
 
 ++++
-<titleabbrev>Resource guide</titleabbrev>
+<titleabbrev>Logs</titleabbrev>
 ++++
 
-In this guide, you'll find resources on sending log data to {es}, configuring your logs, and analyzing your logs.
+Logs are an important tool for ensuring the performance and reliability of your applications and infrastructure. They provide important information for debugging, analyzing performance, and managing compliance.
+
+In this guide, you'll find resources for sending log data to {es}, configuring your logs, and analyzing your logs.
 
 [discrete]
 [[logs-getting-started-checklist]]
 == Get started with logs
 
-If you're new to ingesting, viewing, and analyzing logs with Elastic, see <<logs-metrics-get-started, Get started with logs and metrics>> for an overview of adding integrations, installing and running an {agent}, and monitoring logs.
+For a high-level overview on ingesting, viewing, and analyzing logs with Elastic, refer to <<logs-metrics-get-started, Get started with logs and metrics>>.
+
+To get started ingesting, parsing, and filtering your own data, refer to these pages:
+
+* *<<logs-stream>>*: send log files from your system to {es} using a standalone {agent} and configure the {agent} and your data streams using the `elastic-agent.yml` file.
+* *<<logs-parse>>*: break your log messages into meaningful fields that you can use to filter and analyze your data.
+* *<<logs-filter>>*: find specific information in your log data to gain insight and monitor your systems.
```

Comment:
Is this mid-page link intentional?

Reply:
I'm comparing to the links in the TOC
<img width="681" alt="Screenshot 2024-01-03 at 12 09 48 PM" src="https://github.com/elastic/observability-docs/assets/5618806/7e8b32b8-33b8-4f78-8e69-cb9f4340fa13">

Reply:
good spot, that was not intentional. updated now!